CMakeLists.txt

```diff
@@ -137,7 +137,7 @@ include_directories(${DART_INCLUDE_DIRS})
 # Libraries and unit tests.
 #
 # TODO: Add this to each target's DEFINITIONS property.
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Werror -Wno-unused-local-typedefs")
```

Comment:
Why is `-Wno-unused-local-typedefs` necessary? Could we remove the unused `typedef`s instead? 😉